### PR TITLE
Return a file with the paths of the json files with the extracted features

### LINKED ISSUE #206: Get feature extraction output from specimen_ids
Script requested by a user

### PR DESCRIPTION
**Pull request by sgratiy**
Resolves #206

### CODE REVIEW

File: ipfx/bin/get_fx_output.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    from specimens sp
+    join ephys_roi_results err on err.id = sp.ephys_roi_result_id
+    where sp.id = %d
+    """ % specimen_id
+
+    res = lu.query(query)
+    if res:
+        err_dir = res[0]["storage_directory"]
+
+        file_list = glob.glob(err_dir+'*EPHYS_FEATURE_EXTRACTION_*_output.json')
```
Comment (njmei) [inline ipfx/bin/get_fx_output.py]: I would use `os.path.join()` instead of string summation just to be safe.

File: ipfx/bin/get_fx_output.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--specimen_file', type=str, required=True)
+    parser.add_argument('--specimen_output_json_file', type=str, required=True)
+    args = parser.parse_args()
+
+    return vars(args)
+
+
+def main():
```
Comment (njmei) [inline ipfx/bin/get_fx_output.py]: Maybe consider adding this to `setup.py` console scripts? https://python-packaging.readthedocs.io/en/latest/command-line-scripts.html#the-console-scripts-entry-point

Comment (sgratiy) [reply ipfx/bin/get_fx_output.py]: @njmei good point. If @abuchin will be using this actively then I will ad it to setup. For now, I will bookmark your link, thanks.

**Comment by abuchin** [comment]
Approved, long time ago...

**Pull request merged**